tools/tsdb/bloom-tester/tokenizer.go

```diff
@@ -0,0 +1,220 @@
+package main
```

Comment:
With my changes, it became tricky to use v1.BloomTokenizer because it uses v1.Metrics vs the bloom tester uses it's own metrics. 

I choose an easy solution to copy the file. As the project gets in shape, we don't rely on this tool to benchmark blooms anymore.